src/svelte/src/components/dataEditor.svelte

```diff
@@ -774,658 +781,713 @@ limitations under the License.
           scrollSearchResults(false)
         }
         break
+      case MessageCommand.setUITheme:
+        $darkUITheme = msg.data.theme === 2
+        break
     }
   })
 </script>
 
 <svelte:window on:keydown|nonpassive={handleKeybind} />
-<header>
-  <div class="header-container">
-    <fieldset class="box file-metrics">
-      <legend>File Metrics</legend>
-      <div class="flex-container row wrap">
-        <div class="row-item flex-container col">
-          <label for="file_name" class="col-item file-metrics">Path</label>
-          <div id="file_name" class="col-item file-name">{$fileName}</div>
-        </div>
-      </div>
-      <hr />
-      <div class="flex-container row" style="padding-top: 5pt;">
-        <div class="two-row-items flex-container col">
-          <label class="col-item file-metrics" for="disk_file_size"
-            >Disk Size</label
-          >
-          <div class="col-item" id="disk_file_size">{$diskFileSize}</div>
-        </div>
-        <div class="two-row-item flex-container col">
-          <label class="col-item file-metrics" for="computed_file_size"
-            >Computed Size</label
-          >
-          <div class="col-item" id="computed_file_size">
-            {$computedFileSize}
+<body class={$UIThemeCSSClass}>
```

Comment:
Why use a `body` tag?  This is going to be injected into VS Code markup which will already have a body.

Reply:
I needed a way to set the global font colors per scheme but didn't have a way to inject any styling in the `SvelteWebviewInitializer` that contains the UI body. This was the quickest solution that I predict will change with the implementation of https://github.com/ctc-oss/daffodil-vscode/issues/110